HPE3PARPSToolkit-CLI.psm1

```diff
@@ -34380,11 +34379,2491 @@ Function Stop-3parWsapi()
+ Write-DebugLog "Executing Function : Get-3parHealth Command -->" INFO: 
+ 
+ Return $Result
+} ##  End-of Get-3parHealth
+
+##########################################################################
+######################### FUNCTION Remove-3parAlerts #########################
+##########################################################################
+Function Remove-3parAlerts()
+{
+<#
+  .SYNOPSIS
+   Remove-3parAlerts - Remove one or more alerts.
+
+  .DESCRIPTION
+   The command removes one or more alerts from the system.
+
+  .EXAMPLE
+
+  .PARAMETER  Alert_ID
+	Indicates a specific alert to be removed from the system. This specifier can be repeated to remove multiple alerts. If this specifier is not used, the -a option must be used.
+  
+  .PARAMETER All
+   Specifies all alerts from the system and prompts removal for each alert.
+   If this option is not used, then the <alert_ID> specifier must be used.
+
+  .PARAMETER F
+   Specifies that the command is forced. If this option is not used and
+   there are alerts in the "new" state, the command requires confirmation
+   before proceeding with the operation.
+
+  .Notes
+    NAME: Remove-3parAlerts
+    LASTEDIT 27-03-2019 11:58:25
+    KEYWORDS: 3parVersion
+  
+  .Link
+    Http://www.hpe.com
+
+ #Requires PS -Version 3.0
+#>
+[CmdletBinding()]
+ param(
+	[Parameter(Position=0, Mandatory=$false, ValueFromPipeline=$true)]
+	[switch]
+	$All,
+
+	[Parameter(Position=1, Mandatory=$false, ValueFromPipeline=$true)]
+	[switch]
+	$F,
+
+	[Parameter(Position=2, Mandatory=$false, ValueFromPipeline=$true)]
+	[System.String]
+	$Alert_ID,
+
+	[Parameter(Position=3, Mandatory=$false, ValueFromPipeline=$true)]
+	$SANConnection = $global:SANConnection
+ )
+
+ Write-DebugLog "Start: In Remove-3parAlerts - validating input values" $Debug 
+ #check if connection object contents are null/empty
+ if(!$SANConnection)
+ {
+	#check if connection object contents are null/empty
+	$Validate1 = Test-ConnectionObject $SANConnection
+	if($Validate1 -eq "Failed")
+	{
+		#check if global connection object contents are null/empty
+		$Validate2 = Test-ConnectionObject $global:SANConnection
+		if($Validate2 -eq "Failed")
+		{
+			Write-DebugLog "Connection object is null/empty or Connection object UserName,password,IPAaddress are null/empty. Create a valid connection object using New-SANConnection" " ERR: "
+			Write-DebugLog "Stop: Exiting Remove-3parAlerts since SAN connection object values are null/empty" $Debug 
+			Return "FAILURE : Exiting Remove-3parAlerts since SAN connection object values are null/empty"
+		}
+	}
+ }
+
+ $plinkresult = Test-PARCli -SANConnection $SANConnection
+ if($plinkresult -match "FAILURE :")
+ {
+   write-debuglog "$plinkresult"
+   Return $plinkresult
+ }
+
+	$Cmd = " removealert "
+
+ if($F)
+ {
+	$Cmd += " -f "
+ }
+ 
+ if($All)
+ {
+	$Cmd += " -a "
+ }
+ elseif($Alert_ID)
+ {
+	$Cmd += " $Alert_ID "
+ }
+ else
+ {
+	Return "Please Select Atlist One from [ All | Alert_ID ]..."
```

Comment:
Correct all the occurrence of the sentence "Please Select **Atlist** One from [ All | Alert_ID ]..."

Reply:
DONE..